src/main/java/frc/robot/subsystems/Swerve/DrivebaseSubsystem.java

```diff
@@ -184,15 +190,29 @@ public Rotation2d getGyroscopeRotation() {
     public void periodic() {
 
         swerveOdometry.update(getGyroscopeRotation(), getModulePositions());
-
+        System.out.println(ll.getRobotPose());
         Optional<EstimatedRobotPose> result = pcw.getEstimatedGlobalPose(swerveOdometry.getEstimatedPosition());
-        /* 
+
         if (result.isPresent()) {
             EstimatedRobotPose camPose = result.get();
             swerveOdometry.addVisionMeasurement(
                     camPose.estimatedPose.toPose2d(),
                     camPose.timestampSeconds);
-        }*/
+        }
+        LimelightResults llresults = LimelightHelpers.getLatestResults("limelight");
+
+        NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+        NetworkTableEntry tl = table.getEntry("tl");
+        NetworkTableEntry cl = table.getEntry("cl");
+        NetworkTableEntry tv = table.getEntry("tv");
+
+        double totalLatency = cl.getDouble(1) + tl.getDouble(1);
+        Pose2d please_work = llresults.targetingResults.getBotPose2d_wpiBlue();
```

Comment:
What's the difference with red vs blue here? Do you need to use the red value when you're on red alliance?